packages/cli/openapi-parser/src/schema/convertSchemas.ts

```diff
@@ -160,9 +161,9 @@ export function convertSchemaObject(
     const mixedGroupName =
         getExtension(schema, FernOpenAPIExtension.SDK_GROUP_NAME) ??
         getExtension<string[]>(schema, OpenAPIExtension.TAGS)?.[0];
-    let groupName = (typeof mixedGroupName === "string" ? [mixedGroupName] : mixedGroupName) ?? [];
+    let groupName: SdkGroupName = (typeof mixedGroupName === "string" ? [mixedGroupName] : mixedGroupName) ?? [];
     if (namespace != null) {
-        groupName = [namespace, ...groupName];
+        groupName = [{ type: "namespace", name: namespace }, ...groupName];
```

Comment:
do you think we can move this logic across operation, httpOperation and webhookOperation to a shared utility?